### ISSUE TITLE
Rails on rails negate the effect of mass

### ISSUE THREAD

**Issue by Dominhiho** [tsunamayo/Skywanderers#143]
Rail movement speed is obviously affected by the mass it is attached to, but if you put a rail on a rail and activate the rail below the speed is not affected. Gif link below to explain this a bit better.
[Gif here](https://i.imgur.com/KBfLuzE.gifv)

**Comment by tsunamayo** [comment]
Yes mass / rail physic is very WIP for now, I will improve that for sure at one point. Thanks!

**Comment by Napther** [comment]
If it is mass effected and this is the bypass then there should be some way for builders to get super fast telescoping rails for say. Shuttle trains on big ships. The option to have fast rails shouldn't be removed, so when the mass and power systems come along it isn't too far off to think super speed rails will cost power to run. Small amount of it. but thwre is little tactical advantage currently from speedy rails bar RP

**Comment by tsunamayo** [comment]
Rail for sure will get a refactoring, I will open a post on the forum to see what you want to do with the system. Also for sure I will do rail dedicated for train, with large and smooth turn, so you can create large train - but that would be a different system.

**Comment by tsunamayo** [comment]
I still need to fix this.
